tests/integration/test_api_gateway.py

```diff
@@ -127,3 +148,38 @@ def forward_request(self, **kwargs):
 
     # clean up
     proxy.stop()
+
+
+def test_api_gateway_lambda_proxy_integration():
+    test_port = 12123
+    backend_url = 'http://localhost:%s%s' % (test_port, API_PATH_LAMBDA_PROXY_BACKEND)
+
+    # create target HTTP backend
+    class TestListener(ProxyListener):
```

Comment:
Can we extract this code into separate methods `setup()` and `teardown()` to avoid duplication. These methods are automatically executed by the test framework before/after running tests in this module: http://nose.readthedocs.io/en/latest/writing_tests.html